src/bindings/rust/wrapper.h

```diff
@@ -62,6 +62,14 @@ struct nixl_capi_xfer_req_s;
 struct nixl_capi_notif_map_s;
 struct nixl_capi_query_resp_list_s;
 
+struct nixl_capi_xfer_telemetry_s {
```

Comment:
Is it really needed? Other structs here are defined as opaque, and only implemented in Rust.
If possible, can remove this one and keep the logic in Rust.